game_sessions/usecase/usecase.go

```diff
@@ -105,6 +112,77 @@ func (a *GameSessionsUseCase) NewSession(ctx context.Context, Casino *models.Cas
 
 	return gameSession, nil
 }
+
+func (a *GameSessionsUseCase) GameAction(ctx context.Context, sessionId uint64, actionType uint16, actionParams []uint32) error {
+	gs, err := a.repo.GetGameSession(ctx, sessionId)
+	if err != nil {
+		return err
+	}
+
+	game, err := a.casinoRepo.GetGame(ctx, gs.GameID)
+	if err != nil {
+		return err
+	}
+
+	bcAction := eos.Action{
```

Comment:
& here would be better for code readability, I think

Reply:
ok